api/src/main/java/org/openmrs/api/impl/VisitServiceImpl.java

```diff
@@ -352,43 +352,46 @@ public VisitAttribute getVisitAttributeByUuid(String uuid) {
 	@Override
 	public void stopVisits(Date maximumStartDate) {
 		String gpValue = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_VISIT_TYPES_TO_AUTO_CLOSE);
+		if (StringUtils.isBlank(gpValue)) {
+    		return;
```

Comment:
@Ankitkumar94 please make sure you formatted the code. this line needs indentation.
also see https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE